### ISSUE TITLE
Constructive Solid Geometry in GPU 

### ISSUE THREAD

**Issue by PrashantKT** [nicklockwood/Euclid#131]
is it possible to calculate expensive work i.e looping clipping on GPU using compute shaders ?

**Comment by nicklockwood** [comment]
Yes, it's certainly possible in theory. This is not really my area of expertise though
